backend/src/services/secret/secret-service.ts

```diff
@@ -1382,6 +1387,36 @@ export const secretServiceFactory = ({
     };
   };
 
+  const getSecretByIdRaw = async ({ secretId, actorId, actor, actorOrgId, actorAuthMethod }: TGetASecretByIdRawDTO) => {
+    const sec = await secretV2BridgeDAL.findOneWithTags({
+      [`${TableName.SecretV2}.id` as "id"]: secretId
+    });
+
+    if (!sec) {
+      throw new NotFoundError({
+        message: `Secret with id '${secretId}' not found`,
+        name: "GetSecretById"
+      });
+    }
+
+    const { shouldUseSecretV2Bridge } = await projectBotService.getBotKey(sec.projectId);
+
+    if (shouldUseSecretV2Bridge) {
+      const secret = await secretV2BridgeService.getSecretById({
+        secret: sec,
+        actorId,
+        actor,
+        actorOrgId,
+        actorAuthMethod
+      });
+
+      return secret;
+    }
+    throw new BadRequestError({
+      message: "Project version not supported. Please upgrade your project."
+    });
+  };
```

Comment:
hmmm... if you're alreading fetching from secrets v2, then v1 will always error with `NotFound` right? might not be necessary to do the `shouldUseSecretV2Bridge` check anymore